src/components/form-elements/draft-js-mention-selector/DraftJSMentionSelectorCore.js

```diff
@@ -67,19 +69,23 @@ class DraftJSMentionSelector extends React.Component<Props, State> {
         contacts: [],
         isDisabled: false,
         isRequired: false,
-        mentionTriggers: ['@', '＠', '﹫'],
```

Comment:
If we do `mentionTriggers: defaultMentionTriggers`, does the logic for regex below need to change?